Onboarding/onboarding-part-2.md

```diff
@@ -1,21 +1,24 @@
-# Onboarding: Pt 2 - Pregnancy Detail
+# Onboarding: Pt 2 - Profile Pregnancy Health
 
-This is the second flow that users interact with during Onboarding
+This is the main onboarding flow that users interact with during onboarding. They are directed here to complete their profile for pregnancy health if they are pregnant, have a partner who is pregnant, or are curios about the content.
 
-This flow is about capturing the details of a pregnancy, asking for the day and month to form an Expected Due Date (EDD).
+All content for this flow is stored in the ContentRepo. This stack uses the ContentRepo API to fetch the content, referencing it by the slug. A list of these slugs can be found at the end of this stack.
 
-The content for this flow, except for navigation items and validation errors at this stage,  is stored in ContentRepo.  The flow uses the ContentRepo API to fetch content, referencing it by slug.
+## Contact fields
 
-A full list of required content can be found in the Content Depencies section at the bottom of this stack.
+* `gender`, This stack sets the gender field for the user. If the user selects `im_pregnant` as their status below it defaults to `female`, otherwise it lets them set it to `male`, `female` or `other`.
+* `edd`, Expected Due Date, gets set after we have the EDD month and day provided by the user.
+* `other_children`, How many other children this user has.
 
-## This stack updates the following contact fields
+## Flow results
 
-* `onboarding_part_2`,  gets set to `incomplete` at the start of this stack, and `complete` at the end
-* `edd`, Expected Due Date, gets set at the end of this stack after we have the EDD month and day provided by the user
+* `pregnancy_status`, One of `im_pregnant`, `partner_pregnant` or `curios`
```

Comment:
Should this not rather be `curious`?

Reply:
Do we need store this on a contact field? Seems like their pregnancy status is something that we probably want to refer to later on. (if it doesn't say it should be stored in a field in the Figma flows, maybe we should advocate for it to be)